docs/advanced-topics/checksum-support.rst

```diff
@@ -45,22 +45,35 @@ the one included within the descriptor file itself.
 
 Workflows
 ---------
-For workflows, Docker image checksums are grabbed on snapshot, but this feature is not as reliable as the other checksum support covered so far. Although we can generally
-provide checksum info for referenced Docker images for CWL, WDL, and NFL, there are some caveats. Most conditions are language
-specific, but for all workflow langagues, the images referenced must be from Quay.io or Docker Hub and they must include a version. The following
-are the known constraints for each language.
+For workflows, Docker image checksums are grabbed on snapshot. However, the Docker images we can retrieve from descriptor files
+are more limited compared to the other checksum support covered so far. Although we can generally provide checksum info for referenced Docker
+images for CWL, WDL, and NFL, there are some caveats. Most conditions are language specific, but for all workflow langagues, the images
+referenced must be from Quay.io or Docker Hub and they must include a version. The following are the known constraints for each language.
 
 .. There is a ticket to expand on when we are not able to parse the docker images. This is only what I'm fairly sure about...
 
 Common Workflow Language
 ^^^^^^^^^^^^^^^^^^^^^^^^
 - Various fields can be used to reference a Docker image, but we only support "dockerPull" for now.
-- "$import" or "$include" can be used to reference a local or https file (which may make reference to an image), but we only recognize local imports in this situation.
+- "$import" or "$include" can be used to reference a local or https CWL descriptor, but we do not check for Docker image references made within files using https.
```

Comment:
> but we do not check for Docker image references made within files using https

Suggest "but we do not check for Docker image references made from descriptors imported via http(s)".

Do they only allow https as opposed to http? If yes, then change my above suggestion to  "...via https". If no, then change "a local or https CWL..." to "a local or http(s) CWL...". We forumulate it this way with WDL/Cromwell.